src/ComposerRequireChecker/DefinedSymbolsLocator/LocateDefinedSymbolsFromASTRoots.php

```diff
@@ -10,7 +10,7 @@
 final class LocateDefinedSymbolsFromASTRoots
 {
     /**
-     * @param Traversable|array[] $ASTs a series of AST roots
+     * @param Traversable $ASTs a series of AST roots
```

Comment:
Same as above:

@weirdan the point of `array[]` is to indicate that each element of `$files` is an `array` - the change is incorrect here